### DISCUSSION TITLE
Any plans for media library support on Linux?

### DISCUSSION THREAD

**Discussion by solstice0** [DeaDBeeF-Player/deadbeef#2618] [Q&A] [answered]
I noticed the media library branch is all objective C. Will the media library be Mac exclusive? Or are there plans for a Linux one as well?

**Comment by Oleksiy-Yakovenko** [comment, marked as answer]
Hello,

The media library plugin is implemented in C and is cross-platform.
However, the UI for media library has to be developed separately for Mac
and Linux, and since I'm a Mac user -- I'm developing it on Mac first.
Eventually, when I'm happy with the result and functionality -- the code
will be converted to GTK and Linux.
The media library feature is not going to be mac-exclusive.
